### ISSUE TITLE
`readline()` example in README.md does not work

### ISSUE THREAD

**Issue by mgkuhn** [JuliaIO/LibSerialPort.jl#52]
How to reproduce with Julia 1.4.0 on Ubuntu Linux 18.04 (x86_64):
```julia
using LibSerialPort
port="/dev/ttyS0"

LibSerialPort.open(port, 115200) do s
    write(s, "input")
    ret = readline(s, 1.0) #try to readline with a 1 second timeout
    @show ret
end
```
This results in
```
ERROR: MethodError: no method matching readline(::SerialPort, ::Float64)
Closest candidates are:
  readline(::IO; keep) at io.jl:454
```

**Comment by ianshmean** [comment]
This method is available on master, but a release has't been made yet

**Comment by ianshmean** [comment]
Now released in 0.4
